<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_showallnews</class>
 <widget class="QWidget" name="admin_showallnews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>مدیریت اخبار و اعلامیه ها</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(110, 110, 170)</string>
  </property>
  <widget class="QTableView" name="tableView_News">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>271</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white
</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>عنوان :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>211</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_subject">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white
</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savechange">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ویرایش</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Post">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>مدیریت اخبار و اعلامیه ها</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;</string>
   </property>
   <property name="text">
    <string>متن:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savechange_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>رفرش جدول</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savechange_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>حذف</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
